main/guides/coreeval/local-testnet.md

```diff
@@ -0,0 +1,33 @@
+# Create a Local Testnet
```

Comment:
We already have a section for this: https://docs.agoric.com/guides/dapps/starting-multiuser-dapps.html